jupyterhub/tests/test_api.py

```diff
@@ -722,16 +723,17 @@ def test_group_delete_users(app):
 
 
 @mark.services
-def test_get_services(app, mockservice):
+def test_get_services(app, mockservice_url):
+    mockservice = mockservice_url
     db = app.db
     r = api_request(app, 'services')
     r.raise_for_status()
     assert r.status_code == 200
 
     services = r.json()
     assert services == {
-        'mock-service': {
-            'name': 'mock-service',
+        mockservice.name: {
```

Comment:
It helped debug a bit because I was having some state leak from one test or another. This was mainly to ensure that a service's name wasn't re-used from one test to the next.